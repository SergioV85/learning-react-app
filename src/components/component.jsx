import React from 'react';

export class Subtitle extends React.Component {
  render() {
    return <h2>This is subtitle, simple component</h2>;
  }
}