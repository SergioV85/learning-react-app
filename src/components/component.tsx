import * as React from 'react';

export class Subtitle extends React.Component {
  public render() {
    return <h2 className="custom-header">This is subtitle, simple component</h2>;
  }
}