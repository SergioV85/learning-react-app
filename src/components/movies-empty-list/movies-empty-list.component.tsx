import * as React from 'react';

export const MoviesEmptyList = () =>
  <div className='c-movies-list c-movies-list--empty'>
    <h1 className='c-movies-list__no-found-title'>No films found</h1>
  </div>;
