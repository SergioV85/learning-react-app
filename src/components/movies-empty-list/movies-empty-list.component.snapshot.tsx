import { shallow } from 'enzyme';
import * as React from 'react';
import { MoviesEmptyList } from './movies-empty-list.component';

describe('Movies Empty List Component Snapshot', () => {
  it('should renders correctly', () => {
    const moviesEmptyListSnapshot = shallow(
      <MoviesEmptyList />,
    );
    expect(moviesEmptyListSnapshot)
      .toMatchSnapshot();
  });
});
