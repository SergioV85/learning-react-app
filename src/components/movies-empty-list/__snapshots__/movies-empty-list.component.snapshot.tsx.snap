// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies Empty List Component Snapshot should renders correctly 1`] = `
<div
  className="c-movies-list c-movies-list--empty"
>
  <h1
    className="c-movies-list__no-found-title"
  >
    No films found
  </h1>
</div>
`;
